test/test_connection.py

```diff
@@ -1,11 +1,17 @@
-import unittest
+import datetime
+import sys
+if sys.version_info > (2, 6):
+    import unittest
+else:
```

Comment:
So the reason I added this was to use `assertGreater` which apparently wasn't in `unittest` for Python 2.6. I now realize why it didn't work on CI. I'll push a change that properly chcks `sys.version_info`.